App.js

```diff
@@ -19,13 +19,13 @@ const AppStack = createStackNavigator(
     DonInfo: DonInfoView,
     EditProfile: EditProfileView,
   }
-)
+);
 
-export default createSwitchNavigator(
+export default createAppContainer(createSwitchNavigator(
```

Comment:
What is the purpose of this changing? Is this due to the SDK update?

Reply:
Yep in one of the SDK version between 27 and 32 they enforced having a container for this.